provider/virtual_network/pktgen_utils.py

```
@@ -0,0 +1,252 @@
+    :param pagesize: Page size specification for kernel package selection
+    :param vm: VM instance for guest installation
+    :param session_serial: Serial session for guest command execution
+    """
+    if pagesize:
+        kernel_ver = "kernel-%s-modules-internal-%s" % (pagesize, ver.split("+")[0])
+    else:
+        kernel_ver = "kernel-modules-internal-%s" % ver
+    cmd_download = "cd /tmp && brew download-build %s --rpm" % kernel_ver
+    cmd_install = "cd /tmp && rpm -ivh  %s.rpm --force --nodeps" % kernel_ver
+
+    utils_misc.cmd_status_output(cmd_download, shell=True)
```

Ruff (0.14.10)

Function call with `shell=True` parameter identified, security issue

(S604)